src/components/pages/Offer/domain/localProvider.js

```diff
@@ -0,0 +1,19 @@
+function doesLastProviderExists(offer) {
```

Comment:
Pourquoi une fonction dans ce cas ci ?
```suggestion
const _doesLastProviderExists(offer) => {
```
Et la mettre en bas.